ddtrace/tracer/remote_config.go

```diff
@@ -298,13 +299,26 @@ func initalizeDynamicInstrumentationRemoteConfigState() {
 			time.Sleep(time.Second * 5)
 			diRCState.Lock()
 			for _, v := range diRCState.state {
+				accessStringsToMitigatePageFault(v.runtimeID, v.configPath, v.configContent)
 				passProbeConfiguration(v.runtimeID, v.configPath, v.configContent)
 			}
 			diRCState.Unlock()
 		}
 	}()
 }
 
+func accessStringsToMitigatePageFault(strs ...string) {
+	pageSize := os.Getpagesize()
+	for i := range strs {
+		for offset := 0; offset < len(strs[i]); offset += pageSize {
+			_ = strs[i][offset]
```

Comment:
I'm guessing this is operating under the assumption that accessing the entry will cause the entire page to load?

Reply:
Correct.